### DISCUSSION TITLE
z_blog博客网站无法安装。

### DISCUSSION THREAD

**Discussion by 365v** [1Panel-dev/1Panel#6877] [Q&A]
### 1Panel 版本

v1.10.20-lts

### 问题描述

硬件设备 玩客云，刷的  海纳思系统。
安装了 1panel。 折腾了好久，终于安装上了。
OpenResty
新建网站可以 显示 默认的探针。、

### 重现步骤

安装z_blog报错。
最后一步，无法创建 sqlite数据库。
我选择了 pdo 和sqlite3 都不行。
没有数据库。
![image](https://github.com/user-attachments/assets/5b505f66-7eef-4a1d-83b7-da0749523ff6)
系统环境
![image](https://github.com/user-attachments/assets/7cf7b749-15e0-4dff-be0a-849940b77116)
![image](https://github.com/user-attachments/assets/5bfe892e-4494-4162-9eb2-1bcc4015c6eb)
![image](https://github.com/user-attachments/assets/9e689a69-c128-43b8-9041-fa0be189a5a3)

**Comment by wanghe-fit2cloud** [comment]
1Panel 采用 Docker 容器化方案部署 PHP 运行环境，建议首先在博客官方查询容器化相关的解决方案。同时，请确认 SQLite 数据库所需安装的具体扩展。

另外，在 “网站设置” -> “网站目录” 页面，点击“运行用户/组”后面的保存按钮，将外部挂载的网站目录权限设置为 1000（容器内运行用户的 UID）。

**Comment by 365v** [comment]
> 在 “网站设置” -> “网站目录” 页面，点击“运行用户/组”后面的保存按钮，将外部挂载的网站目录权限设置为 1000（容器内运行用户的 UID）。

经过这个设置，已经安装成功。
